quilt/test/test_import.py

```diff
@@ -0,0 +1,76 @@
+"""
+Tests for magic imports.
+"""
+
+from datetime import datetime
+import os
+
+from quilt.tools import command
+from quilt.tools.const import PACKAGE_DIR_NAME
+from .utils import QuiltTestCase
+
+class ImportTest(QuiltTestCase):
+    def test_imports(self):
+        mydir = os.path.dirname(__file__)
+        build_path = os.path.join(mydir, './build.yml')
+        command.build('foo/package', build_path)
+
+        # Good imports
+
+        import quilt.data.foo.package
+
+        from quilt.data.foo import package
+        from quilt.data.foo.package import dataframes
+        from quilt.data.foo.package import README
+
+        # Good attributes of imported packages
+
+        package.dataframes
```

Comment:
seems useful to do some type introspection here. we know these aren't throwing but how do we know, e.g., that a group is a group, a df is a df, etc.?